Gemfile.lock

```diff
@@ -460,8 +478,9 @@ DEPENDENCIES
   timecop
   uglifier
   uuidtools
+  webdrivers
   whenever
   wirble
 
 BUNDLED WITH
-   2.2.16
+   2.1.4
```

Comment:
Is bundler also something you'd like to standardize across apps?

Reply:
bundler is in the nix-store as `2.1.4` AFAICS so until the nix channel updates I'd prefer not to have messy workarounds.